drivers/net/ethernet/renesas/rswitch.c

```diff
@@ -3630,25 +3676,28 @@ void rswitch_add_ipv4_forward_all_types(struct l3_ipv4_fwd_param *param, struct
 static struct rswitch_ipv4_route *rswitch_get_route(struct rswitch_private *priv, u32 dst_ip)
 {
 	struct rswitch_ipv4_route *routing_list, *default_route;
+	struct rswitch_device *rdev;
 	struct list_head *cur;
```

Comment:
I've cherry-picked your PR and I've found that "cur" and "i" variables became unused. Please, remove them.